Tests/Functional/FailingTestSuite/main.swift

```diff
@@ -10,9 +10,7 @@
 
 class PassingTestCase: XCTestCase {
     static var allTests: [(String, PassingTestCase -> () throws -> Void)] {
-        return [
-            ("test_passes", test_passes),
-        ]
+        return [("test_passes", test_passes)]
```

Comment:
I would also be in favor of keeping the [trailing comma](https://twitter.com/jaybuff/status/710918302185291776) to reduce the number of lines touched when the list of tests is changed.